From: verdre <verdre@v0yd.nl>
Date: Sat, 20 Oct 2018 15:21:02 +0000
Subject: build: Include params.js in portal-helper gresources

Fix a regression causing the portal helper to crash.
In 94423151b2e1f5e2ee8ccc884c52f50673db5fb9 we moved the dbus interface
descriptions into seperate files which is why we had to include the
fileUtils js module. This module imports the params js module, so add
params.js to the gresources file for the portal helper.

(cherry picked from commit ceed3e07e44f2cd1bfdbf273523abc0bb4bbd8c1)

Applied-upstream: 3.30.2, commit:d2393395fff50deacd46500fa0d152c462a0f4be
---
 js/portal-resources.gresource.xml | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/js/portal-resources.gresource.xml b/js/portal-resources.gresource.xml
index 7961c75..f4858cb 100644
--- a/js/portal-resources.gresource.xml
+++ b/js/portal-resources.gresource.xml
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <gresources>
   <gresource prefix="/org/gnome/shell">
+    <file>portalHelper/main.js</file>
+
     <file>misc/config.js</file>
     <file>misc/fileUtils.js</file>
-    <file>portalHelper/main.js</file>
+    <file>misc/params.js</file>
   </gresource>
 </gresources>
